test/parallel/test-child-process-chunked-stdout.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const stream = require('stream');
+const path = require('path');
+
+const producer = fork(path.join(common.fixturesDir, 'stdout-producer.js'),
+    { silent: true });
```

Comment:
Guess we'll have to manually check on these cases for now. Could probably be solved by a eslint plugin or we try to get such a rule into eslint ourself.